### PR TITLE
remove guppy

### PR DESCRIPTION
**Pull request by yuukiiwa**
Removing guppy due to potential licensing problems. Also nanopore would provide the basecalled fastq together with the fast5 file. Will have to update the subway map.

### CODE REVIEW

File: conf/base.config [hunk 1]
```diff
@@ -8,12 +8,6 @@
 ----------------------------------------------------------------------------------------
 */
 
-// Use this flag if using GPUs with Singularity
-if (params.guppy_gpu || params.deepvariant_gpu) {
-    docker.runOptions      = '--gpus all'
-    singularity.runOptions = '--nv'
-}
-
```
Comment (christopher-hakkaart) [inline conf/base.config:16]: This should be kept for deepvariant

File: nextflow.config [hunk 2]
```diff
@@ -15,22 +15,11 @@ params {
...
     barcode_kit                = null
     barcode_both_ends          = false
     trim_barcodes              = false
-    guppy_config               = null
-    guppy_model                = null
-    guppy_gpu                  = false
-    guppy_gpu_runners          = 6
-    guppy_cpu_threads          = 1
-    gpu_device                 = 'auto'
-    gpu_cluster_options        = null
```
Comment (christopher-hakkaart) [inline nextflow.config:29]: Should also be kept here

File: nextflow_schema.json [hunk 3]
```diff
@@ -91,51 +81,6 @@
...
-                    "type": "string",
-                    "default": "auto",
-                    "description": "Basecalling device specified to Guppy in GPU mode using '--device'.",
-                    "fa_icon": "fas fa-fish"
-                },
-                "gpu_cluster_options": {
-                    "type": "string",
-                    "description": "Cluster options required to use GPU resources (e.g. '--part=gpu --gres=gpu:1').",
-                    "fa_icon": "fas fa-fish"
-                },
```
Comment (christopher-hakkaart) [inline nextflow_schema.json:133]: Keep this too.

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
Looks good. I'll go through the repo for references to basecalling and change the subway map

File: nextflow.config (hunk 2 above)
Comment (yuukiiwa) [reply nextflow.config:29]: I have added this back in

File: conf/base.config (hunk 1 above)
Comment (yuukiiwa) [reply conf/base.config:16]: I have added this back in

File: nextflow.config (hunk 2 above)
Comment (yuukiiwa) [reply nextflow.config:29]: I have added this back in

File: nextflow_schema.json (hunk 3 above)
Comment (yuukiiwa) [reply nextflow_schema.json]: I have added this back in

**Review by christopher-hakkaart — APPROVED** [review]
Still need to tidy up changelog and some references to basecalling but code looks good.

**Pull request merged**